webknossos-tracingstore/proto/SkeletonTracing.proto

```diff
@@ -63,6 +63,7 @@ message SkeletonTracing {
     optional BoundingBox userBoundingBox = 10;
     repeated TreeGroup treeGroups = 11;
     repeated NamedBoundingBox userBoundingBoxes = 12;
+    optional string organizationName = 13;
```

Comment:
yes, because the tracingstore now has to be able to ask wK for the dataset layer propeties, so the organization name is now stored with the tracings. I wanted to do this both in skeleton and volume tracings, for uniformity